packages/databyss-api/src/models/Page.js

```diff
@@ -38,18 +39,26 @@ const PageSchema = new Schema({
 /* eslint-disable prefer-arrow-callback */
 /* eslint-disable func-names */
 PageSchema.method('addEntry', async function(values = {}) {
+  const _entryId = ObjectId().toHexString()
```

Comment:
Do we need to generate the IDs explicitly? I think calling `Entry.create` and `Block.create` will generate them...

Reply:
I can create the `id's` explicitly, or i can create and save the `entry`, create and save the `block`, then I would have to save the `entry` again with the new generated `block._id` since they're both linked.